actions/setup-tools/action.yml

```diff
@@ -28,4 +28,8 @@ runs:
         ) >> "$GITHUB_ENV"
 
     - shell: bash
-      run: sudo apt-get update && sudo apt-get install -y --no-install-recommends jo jq
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y --no-install-recommends jo jq
+        # override k3s-image.json with current one
+        cp k3s-images.json $K3S_IMAGES_JSON
```

Comment:
I don't think this change is desirable: this is failing for you because the published image is out of sync with the repo. The fix for this should be to get the image published.

This change as it stands won't generally work outside of this repo... Will it?

Reply:
Ah good point, I didn't think of this action outside the context of the current repo.

> The fix for this should be to get the image published.

You mean to pull `k3-images.json` via the `sync-k3s-images` recipe?

Reply:
Oh I understand now, you're saying we should just publish this, without the snippet above, and regardless of the CI failure?

Reply:
Ok I've pushed a commit undoing this part.